kpi/views/v2/data.py

```diff
@@ -192,6 +194,12 @@ def edit(self, request, pk, *args, **kwargs):
                                                            params=request.GET)
         return Response(**json_response)
 
+    def get_queryset(self):
+        # This method is needed when pagination is activated and renderer is
+        # `BrowsableAPIRenderer`. Because data comes from Mongo, `list()` and
+        # `retrieve()` don't need Django Queryset, we only need pass.
+        pass
```

Comment:
`return None` is what happens implicitly here; it might be clearer to write that explicitly